README.md

```diff
@@ -22,9 +22,18 @@ Adobe maintains separate JavaScript libraries written with [web components](http
 
 ## Using Spectrum CSS
 
-The preferred method of using Spectrum CSS relies on custom properties to swap out variables for different themes and colorstops. This has the lowest bundle size and the simplest usage, but is incompatible with < IE 11.
+The preferred method of using Spectrum CSS relies on custom properties to swap out variables for different themes and colorstops. This has the lowest bundle size and the simplest usage, but is incompatible with IE 11 and below.
 
-To use Spectrum CSS with IE 11, see the [legacy usage documentation](README-legacy.md).
+### Browser support
+
+The current list of browsers officially supported by Spectrum CSS can be found in the `browserslist` section of the project's [package.json file](https://github.com/adobe/spectrum-css/blob/main/package.json).
+This setting is used by the build tools when the source files are built.
```

Comment:
@jawinn This is great! Should we add a note that consumers should determine what support they want to use for their projects and since we maintain a relatively modern code-base, they can process the provided CSS to meet their own needs easily using a tool of their choice?

Reply:
How is this? Any suggestions on the wording?

We maintain a relatively modern codebase. The current list of browsers officially supported by Spectrum CSS can be found in the `browserslist` section of the project's package.json file. This setting is used by the build tools when the source files are built. If you require additional browser support for your project, the CSS can processed further with your chosen tools.